packages/dds/tree/src/feature-libraries/forest-summary/forestSummarizer.ts

```diff
@@ -90,32 +116,51 @@ export class ForestSummarizer implements Summarizable {
 			);
 			fieldMap.set(key, innerCursor as ITreeCursorSynchronous & ITreeSubscriptionCursor);
 		});
-		const encoded = this.codec.encode(fieldMap, this.encoderContext);
 
+		const forestSummaryBuilder = new SummaryTreeBuilder();
+		// Let the incremental summary builder know that we are starting a new summary. It returns whether
+		// incremental encoding is enabled.
```

Comment:
I think line breaks between sentences is more readable than in the middle of them at least when both result in reasonable line lengths.
```suggestion
		// Let the incremental summary builder know that we are starting a new summary.
		// It returns whether incremental encoding is enabled.
```